index.html

```diff
@@ -5692,50 +5692,50 @@ <h3>Data Theft</h3>
 and access control measures, ensuring that they remain
 inaccessible to unauthorized entities.
         </p>
         <p>
 [=Holders=] are advised to use implementations that appropriately
-encrypt their data both in transit and at rest, and protect sensitive
+encrypt their data in transit and at rest and protect sensitive
 material (such as cryptographic secrets) in ways that cannot be easily
-extracted from hardware devices. Furthermore, it is suggested that
-[=holders=] store and manipulate their data only on devices that they
+extracted from hardware or other devices. It is further suggested that
+[=holders=] store and manipulate their data only on devices they
 control, away from centralized systems, to reduce the likelihood of
-attack on their data, or large-scale theft if an attack is successful.
-Furthermore, [=holders=] are encouraged to rigorously control access to
-their credentials and presentations, allowing access only to those
+an attack on their data or inclusion in a large-scale theft if an attack is
+successful. Furthermore, [=holders=] are encouraged to rigorously control
+access to their credentials and presentations, allowing access only to those
 with explicit authorization.
         </p>
         <p>
-[=Verifiers=] are advised to only ask for data necessary for a particular
-transaction and to not retain any data beyond the needs of any particular
+[=Verifiers=] are advised to ask only for data necessary for a particular
+transaction and to retain no data beyond the needs of any particular
 transaction.
         </p>
         <p>
-Regulators are advised to rethink audit requirements such that more
-privacy-preserving mechanisms can be used to achieve similar levels of
-enforcement and audit capabilities. For example, audit-focused regulations
-that insist on collection and long-term retention of personally identifiable
-information can cause harm to individuals and organizations if that same
-information is compromised and accessed by an attacker. The technologies
-described by this specification enable [=holders=] to more-readily prove
-properties about themselves and others, reducing the need for long-term data
+Regulators are advised to reconsider audit requirements such that mechanisms
+that better preserve privacy can be used to achieve similar enforcement and
+audit capabilities. For example, audit-focused regulations that insist on the
+collection and long-term retention of personally identifiable information can
+cause harm to individuals and organizations if that same information is later
+compromised and accessed by an attacker. The technologies
+described by this specification enable [=holders=] to prove properties about
+themselves and others more readily, reducing the need for long-term data
 retention by [=verifiers=]. Alternatives include keeping logs that the
 information was collected and checked, as well as random tests to ensure
 that compliance regimes are operating as expected.
```

Comment:
Rewrapping makes this look like a larger change than it is, which is the
insertion of `existing` in the first line.
```suggestion
Regulators are advised to reconsider existing audit requirements such that
mechanisms that better preserve privacy can be used to achieve similar
enforcement and audit capabilities. For example, audit-focused regulations
that insist on the collection and long-term retention of personally
identifiable information can cause harm to individuals and organizations
if that same information is later compromised and accessed by an attacker.
The technologies described by this specification enable [=holders=] to
prove properties about themselves and others more readily, reducing the
need for long-term data retention by [=verifiers=]. Alternatives include
keeping logs that the information was collected and checked, as well as
random tests to ensure that compliance regimes are operating as expected.
```